hazelcast-integration/openshift/hazelcast-cluster/hazelcast-enterprise-openshift-centos/hazelcast.yaml

```diff
@@ -12,7 +12,7 @@ labels:
 parameters:
 - name: HAZELCAST_IMAGE
   description: "Defines the location of Hazelcast image"
-  value: hazelcast/hazelcast-enterprise:3.12
```

Comment:
This image is alpine based but we call this file centos. I don't think we should keep centos named folder anymore.

Reply:
done